cmd/test.go

```diff
@@ -69,7 +69,7 @@ func newTestCmd(cmds ...*cobra.Command) *cobra.Command {
 
 func bindTestFlags(cmd *cobra.Command, config *testConfig) {
 	cmd.Flags().StringVar(&config.OutputJSON, "output-json", "", "File path to which output can be written in JSON format.")
-	cmd.Flags().StringSliceVar(&config.TestCases, "test-cases", nil, fmt.Sprintf("List of comma separated names of tests to be exeucted. Available tests are: %v", listTestCases(cmd)))
+	cmd.Flags().StringSliceVar(&config.TestCases, "test-cases", nil, fmt.Sprintf("List of comma separated names of tests to be executed. Available tests are: %v", listTestCases(cmd)))
```

Comment:
This change requires `docs/configuration.md` to be updated as well, please check that file and see how to update it by running the corresponding test. Otherwise CI will keep failing.